src/Entity/SecurityAdvisory.php

```diff
@@ -249,11 +245,6 @@ public function calculateDifferenceScore(RemoteSecurityAdvisory $advisory): int
         return $score;
     }
 
-    public function hasPackagistAdvisoryId(): bool
-    {
-        return (bool) $this->packagistAdvisoryId;
-    }
```

Comment:
@glaubinix anything speaking against the changes I did here?

Reply:
If I remember correct, then this was added to generate an advisory ID for all existing advisories. Should be ok to remove now we added one for all advisories.

Reply:
Right that matches my understanding, thanks